### PR TITLE
feat(SWA): add error logs for some network requests

### PR DESCRIPTION
**Pull request by kiryl-zubarau**
This PR adds logs for some network requests.

### CODE REVIEW

**Review by AlicanAkyuz — APPROVED** [review]

File: src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/ContactInformation.tsx [hunk 1]
```diff
@@ -83,7 +86,7 @@ export const ContactInformation: React.FC<ContactInformationProps> = ({
 
         router.push(`/consign/submission/${submission?.id}/thank-you`)
       } catch (error) {
-        console.log(error)
+        logger.error("Submission not submitted", error)
```
Comment (patrinoua) [inline src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/ContactInformation.tsx:89]: Maybe rephrase to "Error during Submission" or "Submission error"?

File: src/v2/Apps/Consign/Routes/SubmissionFlow/Utils/fileUtils.ts [hunk 2]
```diff
@@ -114,7 +114,7 @@ export const uploadPhoto = async (
       },
     })
   } catch (error) {
-    logger.error("Consign submission operation error", error)
+    logger.error("Photo upload failed", error)
```
Comment (patrinoua) [inline src/v2/Apps/Consign/Routes/SubmissionFlow/Utils/fileUtils.ts:117]: "Error during Photo Upload"

**Review by patrinoua — APPROVED** [review]
Left some minor phrasing issues, but looks great otherwise!

**Comment by kiryl-zubarau** [comment]
@AlicanAkyuz I'm going to merge the PR since the failed test it's a problem of [weekly staging sync](https://github.com/artsy/force/pull/9162#issuecomment-1008427586)
